pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -161,31 +165,41 @@ func (vm *windows) Configure() error {
 }
 
 func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
-	// Check if the hybrid-overlay is running
-	_, err := vm.Run("Get-Process -Name \""+HybridOverlayProcess+"\"", true)
-
-	// err being nil implies that hybrid-overlay is running.
-	if err == nil {
-		// Stop the hybrid-overlay
-		stopCmd := "Stop-Process -Name \"" + HybridOverlayProcess + "\""
-		out, err := vm.Run(stopCmd, true)
-		if err != nil {
-			log.Info("unable to stop hybrid-overlay", "stop command", stopCmd, "output", out)
-			return errors.Wrap(err, "unable to stop hybrid-overlay")
-		}
+	// Ensure kubelet service is running before configuring hybrid-overlay-node Windows service
```

Comment:
I would think this would be expressed using a dependency.

Reply:
yes removed this